arch/arm/src/a1x/a1x_pio.c

```diff
@@ -90,12 +90,11 @@ static inline int a1x_pio_pin(pio_pinset_t cfgset)
  ****************************************************************************/
 
 #ifdef CONFIG_A1X_PIO_IRQ
-static int a1x_pio_interrupt(int irq, void *context)
+static int a1x_pio_interrupt(int irq, FAR void *context, FAR void *arg)
```

Comment:
let's remove FAR, I have patch to remove FAR from all arch/arm.

Reply:
Done